src/selectors.js

```diff
@@ -1,10 +1,19 @@
-import { at } from 'lodash'
+const getFromPath = (data, path) => {
+  const subs = path.split('.')
+  let curr = data
+
+  subs.forEach((sub) => {
+    curr = curr && curr[sub]
+  })
+
+  return curr
+}
```

Comment:
Using `reduce` here is more efficient (I don't reaaly understand why...)

```es6
  const getFromPathReduce = (data, path) => {
    const subs = path.split('.')
    return subs.reduce(
      (curr, sub) => curr && curr[sub], 
      data,
    )
  }
```

[benchmark here](https://jsperf.com/foreach-vs-reduce-k-redux-factory)